explainer.md

```diff
@@ -38,6 +47,10 @@ Firefox) downloads content and uses it. An **intermediate** (like Fastly, the
 AMP cache, or old HTTP proxies) downloads content from its author (or another
 intermediate) and forwards it to a client (or another intermediate).
 
+When an HTTP exchange is encoded into a resource, the resource can be fetched
+from a **physical URL** that is different from the **logical URL** of the
+encoded exchange.
```

Comment:
This terminology feels a bit generic; could we come up with something a bit more specific?

Reply:
I don't have any better terms offhand, but I'll look for some.